### ISSUE TITLE
[fix] aria controls syntax in auth page template stories file

### ISSUE THREAD

**Issue by shkeating** [trussworks/react-uswds#2606]
**ReactUSWDS Version: 5.4.0 & USWDS Version: 3.6.0**

# Describe the bug

There is incorrect ARIA syntax in the `aria-controls` attribute is invalid on the `Show password` link found in the Storybook example for the Sign In Page Template. 

If you're new to [storybook](https://storybook.js.org/), it is a tool we use to display the components for this project and is a part of our documentation. 

This issue is in a **stories** file, which is where we output our components to display in the project storybook. This does not seem to be an issue within a component itself.

[The example in the USWDS docs](https://designsystem.digital.gov/templates/authentication-pages/sign-in/) shows different syntax in the code, which seems to be correct.

# To Reproduce

1. Go to [the react-uswds page template for authentification](https://trussworks.github.io/react-uswds/?path=/story/page-templates-authentication-pages--sign-in)
2. Click on the `Open canvas in a new page` icon in the top right corner (see image below)
<img width="165" alt="image" src="https://github.com/trussworks/react-uswds/assets/59394696/7b3659e7-73b8-4dc6-9d57-8a0da631d94f">

3. Run automated accessibility checker of choice on the page that opens to check for ARIA syntax issues
4. See test failure linked to the `Show Password` link

# Expected behavior

We should not see an ARIA syntax issue related to the aria controls on the Show password link. ARIA usage needs to be correct to ensure our friends who use assistive technology are provided with an equivalent experience when using pages that refer to this storybook example to be built.

# Additional context

- more info about requirement [Ensures all ARIA attributes have valid values](https://dequeuniversity.com/rules/axe/4.7/aria-valid-attr-value?application=AxeFirefox)
- [Sign In Page Template in USWDS](https://designsystem.digital.gov/templates/authentication-pages/sign-in/)
- path to file to be edited: `/src/stories/templates/signin.stories.tsx`

# Screenshots

<img width="1177" alt="image" src="https://github.com/trussworks/react-uswds/assets/59394696/46551fec-3728-47d9-958e-808ee4989bee">

**Comment by vinodkv2511** [comment]
Hi @shkeating - I would like to give it a shot. Will put up a PR soon.
I just looked at the DOM in storybook and looks like the ids used in aria-controls are not present in DOM.
